src/main/java/de/tum/in/www1/artemis/web/rest/AssessmentResource.java

```diff
@@ -36,4 +36,23 @@ public AssessmentResource(AuthorizationCheckService authCheckService, UserServic
         }
         return null;
     }
+
+
+    void checkAuthorization(Exercise exercise) {
+        validateExercise(exercise);
+        User user = userService.getUserWithGroupsAndAuthorities();
+        if (!authCheckService.isTeachingAssistantInCourse(exercise.getCourse(), user) &&
```

Comment:
Isn't there a method isAtLeastTeachingAssistantInExercise(...) or isAtLeastTeachingAssistantInCourse(...) that you can use here?

Reply:
Also add some java doc please

Reply:
yep you are right changed it aswell in the method i copied the code from. I also added the desired Java doc